packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,179 @@
+import HomeIcon from "@mui/icons-material/Home";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { styled } from "@mui/material/styles";
+import React, { ReactElement } from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import StyledTooltip from "./OSCALStyledTooltip";
+import {
+  OSCALSystemImplementationTableTitle,
+  StyledHeaderTableCell,
+  StyledTableRow,
+  StyledTableHead,
+} from "./OSCALSystemImplementationTableStyles";
+import { Property } from "@easydynamics/oscal-types";
+import { NIST_DEFAULT_NAMESPACE, namespaceOf } from "./oscal-utils/OSCALPropUtils";
+import { NotSpecifiedTypography } from "./StyledTypography";
+
+const OSCALPropertiesButton = styled(Button)(
+  ({ theme }) => `
+    color: ${theme.palette.primary.main};
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+  `
+) as typeof Button;
+
+/**
+ * Helper to sort namespaces.
+ *
+ * @param propA The first property to compare
+ * @param propB The second property to compare
+ * @returns A positive number if A is before B alphabetically ot a negative number if B is before A
+ *   alphabetically.
+ */
+function byName(propA: Property, propB: Property): number {
+  return propA.name.localeCompare(propB.name);
+}
+
+interface OSCALPropertyProps {
+  /**
+   * Contains property elements
+   */
+  property: Property;
+}
+
+const OSCALProperty: React.FC<OSCALPropertyProps> = ({ property }) => {
+  const NO_INFORMATION = <NotSpecifiedTypography>Not Specified</NotSpecifiedTypography>;
+
+  return (
+    <StyledTooltip title={property.remarks ?? ""}>
+      <StyledTableRow key={property.uuid}>
+        <TableCell>{property.name ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.class ?? NO_INFORMATION}</TableCell>
+        <TableCell>{property.value ?? NO_INFORMATION}</TableCell>
+      </StyledTableRow>
+    </StyledTooltip>
+  );
+};
```

Comment:
I am concerned that our only option to represent a single Property is to have it as a table row... I suppose it's probably fine because I can't think of another place where we may need it...

Reply:
From looking at the many places in the spec where properties appear, it seems like they appear in a very similar manner, which makes sense to have properties appear as a row in a table. Although, we could instead call this `OSCALPropertyRow` instead if that makes more sense?

Reply:
I guess my thought is that it shouldn't be a separate component -- it cannot be displayed outside `OSCALProperties` which is a table; so it's very dependent on that representation. But I guess since it's not exported, then it's likely fine.